modules/react/button/lib/DeleteButton.tsx

```diff
@@ -1,75 +1,54 @@
-  Themeable,
-  EmotionCanvasTheme,
-  createComponent,
-} from '@workday/canvas-kit-react/common';
 
-export interface DeleteButtonProps extends Themeable, GrowthBehavior, BaseButtonProps {
-  /**
-   * The size of the Button.
-   */
-  size?: 'small' | 'medium' | 'large';
-  children?: React.ReactNode;
-}
+import {buttonVars, ButtonContainerProps} from './BaseButton';
+import {createComponent} from '@workday/canvas-kit-react/common';
+import {createStyles, cssVar} from '@workday/canvas-kit-styling';
+import {base, brand, system} from '@workday/canvas-tokens-web';
+import {Button} from './Button';
 
-const getDeleteButtonColors = ({
-  canvas: {
-    palette: {error: themeError},
-  },
-}: EmotionCanvasTheme): ButtonColors => ({
-  default: {
-    background: themeError.main,
-    label: themeError.contrast,
-  },
-  hover: {
-    background: themeError.dark,
+/**
+ * Extends all the style properties from Box to our buttons as well as props from ButtonContainerProps.
+ * We omit `ref` since all of our buttons use `createComponent` and already give access to `ref`.
+ * Use this type to extend and customize any one off buttons that you want full control over styling.
+ */
+export interface DeleteButtonProps extends Omit<ButtonContainerProps, 'ref'> {}
+
+const deleteStyles = createStyles({
+  [buttonVars.default.background]: cssVar(brand.error.base),
+  [buttonVars.default.border]: 'transparent',
+  [buttonVars.default.borderRadius]: cssVar(system.shape.circle),
+  [buttonVars.default.label]: cssVar(brand.error.accent),
+  '&:hover, &.hover': {
+    [buttonVars.hover.background]: cssVar(brand.error.dark),
+    [buttonVars.hover.border]: 'transparent',
+    [buttonVars.hover.label]: cssVar(brand.error.accent),
   },
-  active: {
-    background: themeError.darkest,
+  '&:focus-visible, &.focus': {
+    [buttonVars.focus.background]: cssVar(brand.error.base),
+    [buttonVars.focus.border]: 'transparent',
+    [buttonVars.focus.label]: cssVar(brand.error.accent),
+    [buttonVars.focus.boxShadowInner]: cssVar(base.frenchVanilla100),
+    [buttonVars.focus.boxShadowOuter]: cssVar(brand.common.focusOutline),
   },
-  focus: {
-    background: themeError.main,
+  '&:active, &.active': {
+    [buttonVars.active.background]: cssVar(brand.error.darkest),
+    [buttonVars.active.border]: 'transparent',
+    [buttonVars.active.label]: cssVar(brand.error.accent),
   },
-  disabled: {
-    background: themeError.light,
-    opacity: '1', // allows for overriding the default opacity of 0.4
+  '&:disabled, &:active:disabled, &:focus:disabled, &:hover:disabled': {
+    [buttonVars.disabled.background]: cssVar(brand.error.light),
+    [buttonVars.disabled.label]: cssVar(brand.error.accent),
+    opacity: 1,
   },
 });
 
-/**
- * Use sparingly for destructive actions that will result in data loss, can’t be undone, or will
- * have significant consequences. They commonly appear in confirmation dialogs as the final
- * confirmation before deleting.
- */
```

Comment:
I think we want this comment back? above `export const DeleteButton...`

Reply:
Yeah definitely. Re-added.